modules/nw-metallb-troubleshoot-bgp.adoc

```diff
@@ -16,7 +15,7 @@ As a cluster administrator, if you need to troubleshoot BGP configuration issues
 
 .Procedure
 
-. Display the names of the `speaker` pods:
+. Display the names of the pods in the `metallb-system` namespace:
```

Comment:
in here should be better to list only the frr-k8s pods, you can use the below command
```
oc -n metallb-system get pods -l component=frr-k8s
```